iroh/src/config.rs

```diff
@@ -58,6 +60,7 @@ impl From<&IrohPaths> for &'static str {
             IrohPaths::BaoFlatStorePartial => "blobs-partial.v0",
             IrohPaths::DocsDatabase => "docs.redb",
             IrohPaths::Console => "console",
+            IrohPaths::PeerData => "peers.postcard",
```

Comment:
I'd rather do sth like `peers.bin`, but more a personal preference. Maybe we should include a version number?

Reply:
since we have already introduced strum, I'll move that test to iterate the values so that future paths are automatically added.

About the version number, what would be the purpose of this? is it for us to understand what info is in there? otherwise we don't store or read a peer data version that would help in an upgrade for example. Maybe we can tackle upgrading a peer data format version once it changes

the `.postcard` was more of a hunch I guess, I'm fine with any

Reply:
this is done in https://github.com/n0-computer/iroh/pull/1507